example.config.nix

```diff
@@ -0,0 +1,22 @@
+# This is an example of the nix config file
+# Name you config file `config.nix`
+
+{
+
+  # This is an array of the agents you need
+  # Add any number of agents you want to spin up on the same conductor
+
+  agents =[{
+    id = "agent-1-id";
+    name = "Agent 1";
+    keystore_file = "<Path to keystore file>";
+    public_address = "<public key>";
+  }
+  {
+    id = "agent-2-id";
+    name = "Agent 2";
+    keystore_file = "<Path to keystore file>";
+    public_address = "<public key>";
+  }];
+
```

Comment:
I like this option to customize or add additional agent.  :+1:

Reply:
Also, you had mentioned your intention to have nix build a default agent in addition to this agent config.  I think this is a great idea and would love to see this included in this PR.

Reply:
We can update this in the nixification process